lib/preload.js

```diff
@@ -251,11 +265,19 @@ preload.CONTAINER_NAME = 'resin-image-preloader'
 
 /**
  * Build the preloader docker image
+ * @param {Object} docker - dockerode instance
+ * @param {Object} visuals - resin-cli-visuals (optional)
  * @returns {Stream}
  */
-preload.build = (docker) => {
+preload.build = (docker, visuals) => {
   const stream = new streamModule.PassThrough()
   const files = ['Dockerfile', 'requirements.txt', 'src/preload.py']
+  let progress
+  let matches
```

Comment:
Is there a good reason to hoist this `matches` here? You could declare it inside its `if` below and it should work fine.